packages/lit-ssr/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "lit-ssr",
+  "type": "module",
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "(cd node_modules/template-shadowroot && npm run build) && tsc",
```

Comment:
For now, I'm depending on the github repo for `template-shadowroot` and building it package from source as part of the lit-ssr `build` script since it's not released. We should discuss whether to publish a release yet or not.